libs/expo/betterangels/src/lib/hooks/californiaIdValidation/useCaliforniaIdValidation.ts

```diff
@@ -0,0 +1,24 @@
+import { useClientProfilesQuery } from '../../screens/Clients/__generated__/Clients.generated';
```

Comment:
maybe this needs its own query. something that _only_ returns the id

Reply:
@tglaz I added a slim query just for this. not sure if it should be colocated with the hook 🤷‍♂️

Reply:
I'd go with whatever we're doing today I suppose
I'm not used to doing that, but I realize there are some graphQl conventions etc that would say yes.